website/docs/install.md

```diff
@@ -28,7 +28,7 @@ For either installation method, make sure you have cluster admin permissions:
 If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/deploy/gatekeeper.yaml
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/v3.16.0-beta.0/deploy/gatekeeper.yaml
```

Comment:
This also shouldn't change IMO. I think the resolution is to manually change here and then I think #3258  should resolve this automatically for future releases.